source/documentBase.py

```diff
@@ -138,6 +147,94 @@ def _getNearestTableCell(self, tableID, startPos, origRow, origCol, origRowSpan,
 				destCol+=1 if movement=="next" else -1
 		raise LookupError
 
+	def _probeTableDimension(self, tableID, startPos, origRow, origCol, axis):
+		"""
+		Measures the dimension (width or height) of current table along L{axis} by repeatedly trying 
+		to call L{_getTableCellAt} with ever increasing index along L{axis} to determine the largest possible value
+		for which L{_getTableCellAt} doesn't raise L{LookupError}.
+		@param tableID: the ID of the table
+		@param startPos: the position in the document to start searching from.
+		@type startPos: L{textInfos.TextInfo}
+		@param origRow: the row number of the starting cell
+		@type origRow: int
+		@param origCol: the column number  of the starting cell
+		@type origCol: int
+		@param axis: the axis of movement ("row" or "column")
+		@type axis: string

```

Comment:
While this should not block this PR ideally both axes and directions  of movement should be specified as an enum.

Reply:
Already implemented in #13345.